priv/migrations/add_railway_ipc_published_messages.rb

```diff
@@ -2,10 +2,10 @@ class AddRailwayIpcPublishedMessages < ActiveRecord::Migration
   def change
     create_table :railway_ipc_published_messages, id: false do | t |
       t.uuid :uuid, null: false
-      t.string :message_type, null: false
+      t.string :message_type
```

Comment:
Ahhh, is this so we can find or create by UUID?

Reply:
it's more based on the fact that we just want to limit possible errors of messages being deleted. A status is still required so that still limits the possibility of using `#find_or_create_by(uuid: <UUID>)`. Opting into only using UUID and status for required fields as we set that in our code. Message Type and Encoded Message are a bit of our control through the code flow. 

This is also just to match the null constraint requirements of the `consumed_messages` table in the rest of the codebase.